utils/manifests/utils.go

```diff
@@ -280,3 +280,38 @@ func deleteFile(path string) error {
 	}
 	return nil
 }
+
+//While going from Capital letter to small, insert a whitespace before the capital letter.
+//While going from small letter to capital, insert a whitespace after the small letter
```

Comment:
Excellent.

Reply:
//While going from Capital to Capital, do nothing.

Reply:
Yupss, that is inbuilt logic

Reply:
Do nothing also, when small to small